tests/fixtures/MakeResetPassword/config/bundles.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
+    Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],
+    Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
+    SymfonyCasts\Bundle\ResetPassword\SymfonyCastsResetPasswordBundle::class => ['all' => true],
+];
```

Comment:
This shouldn't be needed now: the test will install the bundle, and its recipe will update this.

Same below for the `reset_password.yaml` file - the recipe will add that.